README.rst

```diff
@@ -197,6 +197,10 @@ Or to install a specific release:
 
     pip install -U pymodbus==X.Y.Z
 
+You can also use Docker to run a local image with the package installed on the image:
+
+    docker pull riptideio/pymodbus
```

Comment:
I am not sure that is correct, with the user I added as secret owner will be janiversen.

Reply:
oh... ideally the docker namespace is the same as the one in github. 

If that's not possible sure, I can update to use your own account. But it creates a sense of discomfort when the namespace in docker is different to the "vendor"/package-namespace.

I can update though if you think that's the way to go. Your call.

Reply:
To be clear, docker hub allows multiple collaborators for the same project/repository.

Having your user as a credential is fine and doesn't necessarily implies that the docker hub has to be under `janiversen`. You could still have `janiversen` as a username under the `riptideio` namespace